components/script/dom/htmlimageelement.rs

```diff
@@ -1453,8 +1453,8 @@ impl VirtualMethods for HTMLImageElement {
     fn attribute_mutated(&self, attr: &Attr, mutation: AttributeMutation) {
         self.super_type().unwrap().attribute_mutated(attr, mutation);
         match attr.local_name() {
-            &local_name!("src") => self.update_the_image_data(),
-            &local_name!("srcset") => self.update_the_image_data(),
+            &local_name!("src") | &local_name!("srcset")  |
+            &local_name!("width") | &local_name!("crossorigin") => self.update_the_image_data(),
```

Comment:
This should include sizes according to https://html.spec.whatwg.org/multipage/images.html#reacting-to-dom-mutations